code/modules/mob/mob_grab.dm

```diff
@@ -296,7 +296,9 @@
 		assailant.next_move = world.time + 10
 		if(!affecting.get_organ_slot("breathing_tube"))
 			affecting.AdjustLoseBreath(1)
-		affecting.setDir(WEST)
+
+		if (!affecting.buckled)
```

Comment:
Can you check what would happen if you grab somebody while they are buckled and then unbuckle them?

Reply:
It's visually the same because the direction is set to the assailant's direction/south by `adjust_position` which is called by the grab's `process` anyway: https://github.com/ParadiseSS13/Paradise/blob/a67d684b010ca2f1d8f0da01be98c19d84762069/code/modules/mob/mob_grab.dm#L214-L223

That line that makes the mob face west should probably be removed anyway because it doesn't make sense for them to face west if you're strangling from the north or something. What do you think?

Reply:
Seems a bit redundant there indeed. Especially since it calls `adjust_position` afterwards